src/tests/test_filesystem_mounting.py

```diff
@@ -0,0 +1,181 @@
+from pytest_pyodide import run_in_pyodide
+
+
+@run_in_pyodide
+def test_filesystem_mounting(selenium):
+    import asyncio
+    from string import Template
+
+    from pyodide.code import run_js
```

Comment:
As this test starts new Pyodide instances, `run_in_pyodide` that runs the code under the running Pyodide instance would be redundant.

You can use `selenium_standalone_noload` fixture for this purpose, that does not initialize the Pyodide instance. The following test would be a good reference: https://github.com/pyodide/pyodide/blob/de16aad02c62ed9798cc4ea7bb85a6489d53dba1/src/tests/test_pyodide.py#L1316-L1329